trino/base/hive-metastore-dc.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hive-metastore
+  labels:
+    instance: dev-presto
+    role: hive-metastore
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      instance: dev-presto
+      role: hive-metastore
+  template:
+    metadata:
+      labels:
+        instance: dev-presto
+        role: hive-metastore
+    spec:
+      containers:
+        - resources:
+            requests:
+              cpu: 1
+              memory: 4Gi
+            limits:
+              cpu: 1
+              memory: 4Gi
+          terminationMessagePath: /dev/termination-log
+          name: hive-metastore
+          command:
+            - /opt/bin/start-hive-metastore.sh
+          livenessProbe:
+            exec:
+              command:
+                - bash
+                - '-c'
+                - netstat -lpn | grep 9083
+            initialDelaySeconds: 30
```

Comment:
I believe all of these probe timeout/threshold settings are set at default values. If that's the case, we don't need to include them here.